packages/instantsearch.js/src/lib/__tests__/server.test.ts

```diff
@@ -99,7 +99,7 @@ describe('getInitialResults', () => {
     const search = instantsearch({
       indexName: 'indexName',
       searchClient: createSearchClient(),
-    });
+    }).addWidgets([connectSearchBox(() => {})({})]);
```

Comment:
This is required because we now only trigger `search` when there are `search` widgets.
It was already the case in a real-life app anyway.